OracleDatabase/RAC/OracleRealApplicationClusters/LICENSE

```diff
@@ -1,27 +0,0 @@
-Copyright (c) 2018 Oracle and/or its affiliates.  All rights reserved.
```

Comment:
What's the rationale for deleting the LICENSE file?

Reply:
We already have a LICENSE file in the root of the repository. We don't need multiple copies of the same LICENSE file and this repo does not accept any content that is not covered under the UPL.

Reply:
We already have duplicate LICENSE files in many subfolders pretty much since day one.
If we want to clean that up we should do it consistently across all subfolders, not here and there.

Reply:
Done: https://github.com/oracle/docker-images/pull/2097

Reply:
Awesome, thanks!

Reply:
This has been merged, so the `LICENSE` files should be removed from this PR too.